bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -551,10 +544,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params.c240822.nc</paramfile>
-<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c240814.nc</paramfile>
-<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c240814.nc</paramfile>
-<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c240814.nc</paramfile>
+<paramfile phys="clm6_0" >/glade/campaign/cgd/tss/people/oleson/modify_param/ctsm60_params.c241017.nc</paramfile>
+<paramfile phys="clm5_1" >/glade/campaign/cgd/tss/people/oleson/modify_param/ctsm51_params.c241017.nc</paramfile>
+<paramfile phys="clm5_0" >/glade/campaign/cgd/tss/people/oleson/modify_param/clm50_params.c241017.nc</paramfile>
+<paramfile phys="clm4_5" >/glade/campaign/cgd/tss/people/oleson/modify_param/clm45_params.c241017.nc</paramfile>
```

Comment:
@olyson at this point I will
- [ ] copy the new files to .../lnd/clm2/paramdata and
- [ ] ./rimport them

...and you
- [ ] can change the paths here.